controllers/appwrapper_controller.go

```diff
@@ -57,12 +56,15 @@ var (
 	useMachineSets   bool
 
 	maxScaleNodesAllowed int
-	msLister             v1beta1.MachineSetLister
-	machineLister        v1beta1.MachineLister
 	msInformerHasSynced  bool
 	machineClient        mapiclientset.Interface
 	queueJobLister       v1.AppWrapperLister
 	kubeClient           *kubernetes.Clientset
+	factory              = machineinformersv1beta1.NewSharedInformerFactoryWithOptions(machineClient, resyncPeriod()(), machineinformersv1beta1.WithNamespace(""))
+	informer             = factory.Machine().V1beta1().MachineSets().Informer()
```

Comment:
We should seek to use the controller-runtime / manager client to consume the MachineSet API, instead of managing our own client / informer.